packages/ui/src/components/Avatar.tsx

```diff
@@ -87,7 +87,12 @@ export const ContactAvatar = React.memo(function ContactAvatComponent({
   const contact = useContact(contactId);
   return (
     <ImageAvatar
-      imageUrl={overrideUrl ?? contact?.avatarImage ?? undefined}
+      imageUrl={
+        overrideUrl ??
+        contact?.customAvatarImage ??
```

Comment:
Doesn't the calculated column for `contact.avatarImage` already handle coalescing these values?